lrlex/src/lib/parser.rs

```diff
@@ -1311,4 +1320,16 @@ a\[\]a 'aboxa'
         }
         LRNonStreamingLexerDef::<DefaultLexeme<u8>, u8>::from_str(&src).ok();
     }
+
+    /// Test with various /// [Pattern_White_Space](https://unicode.org/reports/tr31/) separators.
```

Comment:
I'm not sure about the inner `///`?

Reply:
Fixed in https://github.com/softdevteam/grmtools/pull/340/commits/7635044013a546b6c90e145c7cbcc650d3de337a